File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.js
function solution(array) {
    var answer = 0;
    
    for(let i = 0; i < array.length - 1; i++) {
        for(let j = i+1; j < array.length; j++) {
            if(array[i] > array[j]) {
                var temp = array[i];
                array[i] = array[j];
                array[j] = temp;
            }
        }
    }
    
    let i = Math.floor(array.length / 2);
    answer = array[i];
    
    return answer;
}

File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.java
class Solution {
    public int solution(int[] array) {
        int answer = 0;
        
        // 오름차순 정렬 -버블정렬
        for(int i = 0; i < array.length - 1; i++) {
            for(int j = i+1; j < array.length; j++) {
                if(array[i] > array[j]) {
                    int temp = array[i];
                    array[i] = array[j];
                    array[j] = temp;
                }
            }
        }

        // 중앙값 구하기
        int i = array.length / 2;
        answer = array[i];
        
        // 다른 풀이
        // Arrays.sort(array);
        // answer = array[array.length/2];
        
        return answer;
    }
}